### PR TITLE
Tree cutting

### PR DESCRIPTION
**Pull request by KazeEspada**
You can now cut down trees! You can use anything sharp to do so, and it will produce 10 logs. The more force the item has, the faster it cuts.

### CODE REVIEW

**Comment by Alek2ander** [comment]
Things you can cut trees with:
- gar glasses
- scalpels
- broken bottles
- throwing stars
I think you need to add a size minimum as well...

**Comment by ATDoop** [comment]
cutting trees with gar glasses sounds good to me

**Comment by xxalpha** [comment]
Yeah a size minimum would be nice. Broken bottles shouldn't be cutting trees.

**Comment by duncathan** [comment]
Nah that's boring
You can always change the scaling to make weak shit really slow and stronger shit fast

**Comment by KazeEspada** [comment]
Weak shit takes 20 seconds.

**Comment by TheNightingale** [comment]
Whitelist it to axes (fire/energy), hatchets, chainsaws and other stuff you can actually cut down trees with. We don't have many tree-cutting implements anyway, so it won't be too hard. Cutting down a tree with a sharpened herring can't be done.

**Comment by PKPenguin321** [comment]
sure it can

**Comment by TheNightingale** [comment]
I-
Wait, you're the guy who added in sharpening blocks, aren't you?
You literally can cut a tree down with a herring now... okay, whitelist suggestion withdrawn, that's too good to pass up.

**Comment by KazeEspada** [comment]
@TheNightingale WHITELISTS ARE MAJOR NO NO. On a scale from 1 to HORRIBLE. whitelists are BYOND HORRIBLE.

File: code/modules/hydroponics/growninedible.dm [hunk 1]
```diff
@@ -68,7 +68,7 @@
 
 /obj/item/weapon/grown/log/attackby(obj/item/weapon/W, mob/user, params)
 	..()
-	if(istype(W, /obj/item/weapon/circular_saw) || istype(W, /obj/item/weapon/hatchet) || (istype(W, /obj/item/weapon/twohanded/fireaxe) && W:wielded) || istype(W, /obj/item/weapon/melee/energy) || istype(W, /obj/item/weapon/twohanded/required/chainsaw))
```
Comment (KazeEspada) [inline code/modules/hydroponics/growninedible.dm]: This is a fucking whitelist. Look at this monster, it even fucking uses the : operator.

**Comment by Iamgoofball** [comment]
whitelists are shit

long live cutting down trees with sharpened herrings

**Comment by WJohn** [comment]
Could you make it so clicking on a tree while on help intent makes you hug the tree?

Pls.

**Comment by KazeEspada** [comment]
Get your shit tree hugger agenda out of here.

**Pull request merged**